start_pokealarm.py

```diff
@@ -140,9 +141,26 @@ def parse_settings(root_path):
                         help='Maximum number of attempts an alarm makes to send a notification.')
     parser.add_argument('-tz', '--timezone', type=str, action='append', default=[None],
                         help='Timezone used for notifications.  Ex: "America/Los_Angeles"')
+    parser.add_argument('--log_to_file',
+                        help=('Enable logging to file.'),
+                        action='store_true', default=False)
+    parser.add_argument('--log_path',
+                        help=('Defines directory to save log files to.'),
+                        default='logs/')
 
```

Comment:
Please add a short option to both of these config parameters.  

Also please ensure that the argument type, action, and default value are being specified to conform with coding standards.

Reply:
+1, short flags are good.

I feel like two flags feels redundant: is it possible to combine them into a single flag somehow? Just a suggestion.

Reply:
When I combine both parameters to something like `--log /some/dir/pokealarm.log` it will be hard to find a non-ridiculous short option since `-l` is already taken and everything else just doesn't feel right. Also I think that `--log` would be short enough. What do you think?